perfeed/settings/weekly_summary_prompts.toml

```diff
@@ -1,31 +1,64 @@
+- A brief overview of the types of PRs
+- Highlight significant changes or improvements introduced.
+- Mention any noteworthy refactors or architecture changes.
+- Provide the details of pr review process.
 
-- Focus on the high level changes; highlight the most important things that were done.
-- When quoting variables or names from the code, use backticks (`) instead of single quote (').
-- Use bullet points (-) to list things, each in a new line.
 
-An example of the output is below:
-=====
-Here's a summary of the team's pull requests:
-- fix a bug that caused ...
-- add a new feature to help ...
-- refactor some legacy code to make ...
-- document some files ...
-=====
+Example 1:
+John merged 2 PRs this week - one is a documentary/bug fix and the other one is a feature enhancement.
+
+Overview: 
+    - The bug fix addressed inconsistencies in the PRSummarizer output, ensuring it returns a typed response. The enhancement introduced a new base class for git providers, supporting extensibility and async operations.
+---    
+Significant Changes:
+    - Bug Fix: Renamed PRDescription to PRSummary for clarity and updated return types across the codebase.
+    - Enhancement: Added BaseGitProvider to enable support for multiple implementations beyond GitHub and refactored the GitHubProvider to use asynchronous calls for improved concurrency.
+---    
+Refactors/Architecture:
+    - Deprecated the get_patch method in favor of direct API calls to diff_url.
+    - Removed an unused GithubProvider class, streamlining the architecture.
+---    
+Review Process:
+    - For the bug fix, colb157 suggested adding a docstring to the renamed PRSummary class. This was implemented, improving clarity for future developers.
+    - For the enhancement, there was a request from luizz44 to split the async refactor and the addition of the base provider into separate PRs for easier review. The author successfully declined this ask, justifying that the changes were tightly coupled and provided adequate testing documentation to support the combined PR.
+    - Both PRs were merged within ~12 hours and ~27 hours of submission after addressing feedback.
 
-Keep the output concise so it's easy to read and understand.
+Example 2:
+Maria closed 3 PRs this week, focusing on bug fixes and performance improvements.
+
+Overview: 
+    - Maria's work targeted a critical bug causing API timeouts and optimized network API calls for better performance.
+---
+Significant Changes:
+    - Fixed an issue where API calls would fail under high concurrency.
+    - Introduced caching for frequently accessed data, reducing redundant requests.
+---    
+Refactors/Architecture:
+    - Refactored API handler logic to separate concerns, improving maintainability.
+---    
+Related Files:
+    - Modified api/handler.py to include caching logic and updated network request code to handle async calls.
+    - Removed legacy utility methods that were no longer in use.
+---    
+Review Process:
```

Comment:
👍🏼  These are good examples that managers would want to highlight in PR review! I think surfacing these insights creates values for them.